addons/agent_mirrorpeer_controller.go

```diff
@@ -288,12 +302,23 @@ func (r *MirrorPeerReconciler) getS3bucket(ctx context.Context, mirrorPeer multi
 			StorageClassName: namespace + ".noobaa.io",
 		},
 	}
+
 	err := r.SpokeClient.Get(ctx, types.NamespacedName{Name: bucket, Namespace: namespace}, noobaaOBC)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			r.Logger.Info("ObjectBucketClaim not found, will be created", "bucket", bucket, "namespace", namespace)
+		} else {
+			r.Logger.Error("Failed to get ObjectBucketClaim", "error", err, "bucket", bucket, "namespace", namespace)
+		}
+	} else {
+		r.Logger.Info("ObjectBucketClaim retrieved successfully", "bucket", bucket, "namespace", namespace)
+	}
 	return noobaaOBC, err
 }
 
 // enableMirroring is a wrapper function around toggleMirroring to enable mirroring in a storage cluster
 func (r *MirrorPeerReconciler) enableMirroring(ctx context.Context, storageClusterName string, namespace string, mp *multiclusterv1alpha1.MirrorPeer) error {
+	r.Logger.Info("Enabling mirroring on StorageCluster", "storageClusterName", storageClusterName, "namespace", namespace)
```

Comment:
```suggestion
        logger := r.Logger.With("storageClusterName", storageClusterName, "namespace", namespace)
	logger.Info("Enabling mirroring on StorageCluster")
```

Reply:
Not required for this as its a single line function